pom.xml

```diff
@@ -31,26 +31,17 @@
         <organization>Salesforce</organization>
     </developer>
     <developer>
-        <name>Steven Tamm</name>
+        <name>Andrzej Poblocki</name>
         <roles>
             <role>Member</role>
         </roles>
-        <email>stamm@salesforce.com </email>
-        <organization>Salesforce</organization>
-    </developer>
-    <developer>
-        <name>Bo Yang</name>
-        <roles>
-            <role>Member</role>
-        </roles>
-        <email>bo.yang@salesforce.com</email>
+        <email>apoblocki@salesforce.com</email>
         <organization>Salesforce</organization>
     </developer>
   </developers>
 
   <properties>
-    <maven.compiler.source>17</maven.compiler.source>
-    <maven.compiler.target>17</maven.compiler.target>
+    <maven.compiler.release>11</maven.compiler.release>
```

Comment:
The [`release` property](https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#release) should be good enough to build project by targeting Java 11.   Tested w/ JDK11 and 17 compiler to build this project